### PR TITLE
UI custom css

### PR DESCRIPTION
**Pull request by mateuszmidor**
KNTK-221
Review2 comments

### CODE REVIEW

**Review by Fenthick — APPROVED** [review]

**Review by mmac-m3a — COMMENTED** [review]
I'd like my comments to be answered or addressed before merging the pull request.

File: domain/config/config.py [hunk 1]
```diff
@@ -36,3 +37,8 @@ def jitter(self) -> Thresholds:
     def packet_loss(self) -> Thresholds:
         """ Packet loss thresholds, in percents (0-100) """
         pass
+
+    @property
+    def matrix(self) -> Matrix:
```
[the remark this answers is not in the document]
Comment (mateuszmidor) [reply domain/config/config.py:42]: Matrix exposes configuration related to connection matrix

File: domain/config/matrix.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
```
[the remark this answers is not in the document]
Comment (mateuszmidor) [reply domain/config/matrix.py:1]: It may later be extended with cell size, default zoom level, etc

**Review by mmac-m3a — APPROVED** [review]
OK

**Pull request merged**